exercises/hello-world/TUTORIAL.md

```diff
@@ -263,24 +263,20 @@ There are many more exercises you can practice with.  Grab the next one! (exampl
 $ exercism download --exercise=two-fer --track=java
 ```
 
-## Review (and comment on) others' submissions to this exercise
+## Become a mentor
 
 The heart of Exercism are the conversations about coding
-practices.  It's definitely fun to practice, but engaging with others
-both in their attempts and your own is how you get feedback.  That feedback
+practices. It's definitely fun to practice, but engaging with others
+both in their attempts and your own is how you get feedback. That feedback
 can help point out what you're doing well and where you might need to
 improve.
 
 Some submissions will be nearly identical to yours; others will be
-completely different.  Seeing both kinds can be instructive and interesting.
+completely different. Seeing both kinds can be instructive and interesting.
 
-Note that you can only view submissions of others for exercises you have
-completed yourself.  This enriches the experience of reading others' code
-because you'll have your own experience of trying to solve the problem.
-
-Here's an up-to-date list of submissions on the Java track:
-
-http://exercism.io/tracks/java/exercises
+Mentors review submitted solutions for the track they've chosen to mentor
+to help users learn as much as possible. To read more about mentoring and
+to sign up see: http://mentoring.exercism.io/.
```

Comment:
We don't have much documentation talking about mentoring, approving exercises and the track structure. I'm happy to improve/add to this section about how to become a mentor if anyone has any suggestions 🙂 But we should maybe also open an issue for adding some documentation somewhere (not sure where would be best) about how mentoring, approving exercises and the track structure works? Or do we think this is covered by other areas of the exercism website @exercism/java?

Reply:
I am under the impression that the mentoring workflow should be covered by the project as a whole and the specifics of each language should be addressed as amendments. Also, I found the current (end of July) documentation for mentors to be sufficient.

Reply:
Great, thanks @PTrottier! In which case we might not need any more documentation then 🙂